packages/devtools_app/lib/src/ui/icons.dart

```diff
@@ -130,7 +130,7 @@ class CustomIconMaker {
     final widgetTheme = WidgetTheme.fromName(name);
     if (widgetTheme.iconAsset != null) {
       return iconCache.putIfAbsent(name, () {
-        return AssetImageIcon(asset: widgetTheme.iconAsset);
+        return AssetImageIcon(asset: widgetTheme.iconAsset!);
```

Comment:
hoist `widgetTheme.iconAsset` into a local var before the `if (widgetTheme.iconAsset != null)`, then use it both places to avoid a second null check

Reply:
done